src/rendering/RenderUtil.cc

```diff
@@ -2058,6 +2058,29 @@ void RenderUtilPrivate::LowlightNode(const rendering::NodePtr &_node)
   }
 }
 
+/////////////////////////////////////////////////
+void RenderUtil::SetWireBoxScale(const Entity &_entityId,
+    const math::Vector3d &_scale)
+{
+  /// \TODO(anyone) Consider to use this->dataPtr->sceneManager.NodeById()
+  /// when WireBoxes are tracked in the scene manager.
+  if (this->dataPtr->wireBoxes.find(_entityId) ==
+          this->dataPtr->wireBoxes.end())
+  {
+    return;
+  }
+
+  ignition::rendering::WireBoxPtr wireBox = this->dataPtr->wireBoxes[_entityId];
+  if (!wireBox)
+    return;
```

Comment:
It would be helpful to print warnings for these returns, so we can debug situations when the scaling fails.

Reply:
Done in 7d0728a.

Reply:
I might be wrong but Ian recommended this method because the scene manager isn't tracking wireboxes so far.